apps/showcase/src/assets/trainings/sdk/steps/plugins/exercise/app.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, inject, signal } from '@angular/core';
+import { type Pet, PetApi } from 'sdk';
+
+@Component({
+  selector: 'app-root',
+  standalone: true,
+  imports: [],
+  templateUrl: './app.component.html',
+  styleUrl: './app.component.scss'
+})
+export class AppComponent {
+  /** Title of the application */
+  public title = 'tutorial-app';
+
+  private readonly petStoreApi = inject(PetApi);
+  readonly #pets = signal<Pet[]>([]);
```

Comment:
why mixin "TS Private" with "es6 Private" ?

Reply:
To be consistent with the previous steps: https://github.com/AmadeusITGroup/otter/pull/2493#discussion_r1850704571 
I can update to keep using `private` everywhere instead of the `#` prefix